noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/main.nr

```diff
@@ -1,31 +1,68 @@
+// docs:start:token_all
```

Comment:
I copied the standard token contract here and just inserted the refunds functionality. So the diff can be ignored until line 426 where the refunds stuff starts.

Reply:
Thanks for this and similar comments! It would have been good to also have a small comment in the PR description explaining why you chose to do things this way, what needs to happen to merge these, etc.